<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIMain</class>
 <widget class="QMainWindow" name="UIMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="twPages">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="wSearchResults">
       <attribute name="icon">
        <iconset theme="system-search">
         <normaloff/>
        </iconset>
       </attribute>
       <attribute name="title">
        <string>Search Results</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="AnnouncementView" name="lvAnnouncements">
          <property name="contextMenuPolicy">
           <enum>Qt::ActionsContextMenu</enum>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="iconSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="resizeMode">
           <enum>QListView::Adjust</enum>
          </property>
          <property name="uniformItemSizes">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lLegend">
         <property name="text">
          <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: line-through; color:#a0a0a0;&quot;&gt;Ignored&lt;/span&gt; | &lt;span style=&quot; font-weight:600;&quot;&gt;Bookmarked&lt;/span&gt; | &lt;span style=&quot; color:#0000ff;&quot;&gt;Normal&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lStatusBar">
         <property name="text">
          <string notr="true">Status bar</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="dwInputSearch">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>83</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Search Criteria</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="InputSearchWidget" name="iswSearch" native="true"/>
     </item>
     <item>
      <widget class="QWidget" name="wSearch" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>199</width>
            <height>8</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pbSearch">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="tbActions">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>Tools bar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="aSwitchAnnouncementIgnoreState"/>
   <addaction name="aSwitchAnnouncementBookmarkState"/>
   <addaction name="aOpenAnnouncementInNewTab"/>
   <addaction name="aOpenAnnouncementInDefaultBrowser"/>
  </widget>
  <action name="aSwitchAnnouncementIgnoreState">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-clear-list">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Switch the ignore state of  the announcement</string>
   </property>
  </action>
  <action name="aSwitchAnnouncementBookmarkState">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="bookmarks">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Switch the bookmark state of the announcement</string>
   </property>
  </action>
  <action name="aOpenAnnouncementInNewTab">
   <property name="icon">
    <iconset theme="document-open-remote">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Open the announcement in a new tab</string>
   </property>
  </action>
  <action name="aOpenAnnouncementInDefaultBrowser">
   <property name="icon">
    <iconset theme="download">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Open the announcement in the default browser</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InputSearchWidget</class>
   <extends>QWidget</extends>
   <header>widgets/InputSearchWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AnnouncementView</class>
   <extends>QListView</extends>
   <header>widgets/AnnouncementView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
